emission/core/wrapper/common_trip.py

```diff
@@ -0,0 +1,14 @@
+import logging
+import emission.core.wrapper.wrapperbase as ecwb
+
+class CommonTrip(ecwb.WrapperBase):
+    props = {"start_loc" : ecwb.WrapperBase.Access.WORM, # _id of common_place of starting point
+             "end_loc" : ecwb.WrapperBase.Access.WORM, # _id of common_place of ending point
+             "trips" : ecwb.WrapperBase.Access.WORM, # List of trip_ids that are associated with this common trip
```

Comment:
Yeah sorry I meant Naomi's code didn't include that when she packaged the info, I can add that